org.goldcoinproject.goldcoin-qt.json

```diff
@@ -0,0 +1,67 @@
+{
+  "id": "org.goldcoinproject.goldcoin-qt",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "22.08",
+  "command": "goldcoin-qt",
+  "finish-args": [
+    /* Wayland and Xorg support */
+    "--socket=x11",
+    /* Sound */
+    "--socket=pulseaudio",
+    /* OpenGL access. */
+    "--device=dri",
+    /* IPC access. */
+    "--share=ipc",
+    /* Network acess. */
+    "--share=network"
+  ],
+  "modules": [
+    {
+      "name": "goldcoin",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -m 0744 -D -t /app/bin/internal bin/goldcoin-qt"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "only-arches": ["x86_64"],
+          "url": "https://github.com/goldcoin/goldcoin/releases/download/v0.14.9.1/goldcoin-0.14.9.1-x86_64-linux-gnu.tar.gz",
+          "sha256": "a9c6c6d6fdd9f021c1d8e22a18d4c83199f36362bf1579839f14af7c5c456204"
+        }
+      ]
+    },
+    {
+      "name": "misc",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dp -m 644 goldcoin-icon-transparent.svg /app/share/icons/hicolor/scalable/apps/org.goldcoinproject.goldcoin-qt.svg",
+        "install -Dp -m 644 goldcoin-qt.desktop /app/share/applications/org.goldcoinproject.goldcoin-qt.desktop",
+        "desktop-file-edit --set-icon=${FLATPAK_ID} /app/share/applications/org.goldcoinproject.goldcoin-qt.desktop",
+        "install -Dp -m 644 org.goldcoinproject.goldcoin-qt.metainfo.xml /app/share/metainfo/org.goldcoinproject.goldcoin-qt.metainfo.xml",
+        "bash install_wrapper.sh"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "url": "https://raw.githubusercontent.com/expatjedi/flatpak-data/main/goldcoin-qt.desktop",
```

Comment:
so I will reiterated what you have been ignoring. This download the latest version of the file in `main`. Which mean that:

1. it might no longer be there. BUILD WILL FAIL
2. it might have changed, checksum will differe, BUILD WILL FAIL.

Github allow you to get a download link for a specific revision.

Also this should be in your next release so it's part of the build artifact and doesn't need to be downloaded separately, since a destkop app without a .desktop is a bug (app is incomplete)

Reply:
I could move the file to goldcoin's official main repository. The file can be left as it is because it has a fixed content and will never change. It's not something to focus on that much. I made the whole content of my manifest file using another coin's package. I'm sure your intenions are good but wondering if you questioned them that much. Their manifest has no alterations based on your recommendations.

Reply:
then against just change the URL to point to the specific commit. I don't know why you resist so much doing so. You have spend more time argueing reviewers than it would have taken to actually address the issue.

Reply:
There is no issue here and it's not against any policy because peercoin has the same line of codes on flatpak. You guys missed their codebase i guess. Closing this one. No more argue.